app-admin/syslog-ng/syslog-ng-3.15.1.ebuild

```diff
@@ -0,0 +1,165 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit autotools python-single-r1 eutils multilib systemd versionator
```

Comment:
eutils is unused (see split eclasses). multilib is unused. versionator -> eapi7-ver. Sort.